protocols/development-process/work-approach-checklist.md

```diff
@@ -0,0 +1,217 @@
+---
+permalink: /protocols/development-process/work-approach-checklist.html
+---
+
+<link rel="stylesheet" href="roles.css" />
+
+# Work approach checklist
+
+* auto-gen TOC:
+{:toc}
+
+## Introduction
+
+The following constitutes a checklist that should become the natural way to
+resolve issues. It is intended to be exhaustive, so as to cover each base.
+
+It should be noted that often not all steps are applicable for successfully
+resolving a ticket. If so, the respective steps can quickly be checked off as
+N/A. It should also be noted that the list is not intended to be written out in
+such detail that it becomes obstructive of the actual implementation process by
+taking too much time. The list is intended, however, to serve as a guideline for
+what in due time should become the natural approach to resolve tickets.
+
+The following Team Members are recognized in this process:
+
+- <span class="role role-end-user">End User</span>
+
+- <span class="role role-stakeholder">Stakeholder</span>
+
+- <span class="role role-product-owner">Product Owner</span>
+
+- <span class="role role-scrum-master">Scrum Master</span>
+
+- <span class="role role-quality-assurance">Quality Assurance</span>
+
+- <span class="role role-developer">Developer
+
+  - <span class="role role-ux">User eXperience </span>
+  - <span class="role role-graphic-designer">Graphic Designer </span>
+  - <span class="role role-programmer">Programmer</span>
+
+For each step a single role/person is responsible (optionally assisted by Scrum
+Master or QA)
+
+-------------------------------------------------------------------------------
+
+## ![](images/idea.png) Idea Ticket
+
+### 1. **Is the IDEA clear enough to create an IDEA ticket❔**
+
+When an <span class="role role-end-user">End User</span> or stakeholder has
```

Comment:
Originally the "end-user" was added as they do not appear anywhere else in the document. Will remove inline-styling.